misc/gen-syntax.py

```diff
@@ -203,28 +210,27 @@ def pad_prefix(txt):
         if cmd is None:
             continue
         elif cmd in known_words or cmd in commands:
-            logger.info("%s already recorded as something, skipping" % cmd)
+            logger.info("{} already recorded as something, skipping".format(cmd))
         else:
             commands.append(cmd)
     sqfCommand = Definition("sqfCommand", "Function", commands)
     definitions = (sqfCommand,) + WELL_KNOWN
 
     # Write output
-
-    print(HEADER.format(a.version))
+    write_output(output, HEADER.format(args.version))
 
     for defn in definitions:
         members = sorted("\\" + m if m.startswith("|") else m for m in defn.members)
         text = " ".join(members)
-        prefix = pad_prefix("syn keyword     %s " % (defn.name))
-        for line in textwrap.wrap(
-            text, initial_indent=prefix, subsequent_indent=prefix, width=LINE_WIDTH
-        ):
-            print(line)
-        print("")
+        prefix = pad_prefix("syn keyword     {} ".format(defn.name))
+
+        wrap = textwrap.wrap(text, initial_indent=prefix, subsequent_indent=prefix, width=LINE_WIDTH)
```

Comment:
I use [black](https://github.com/psf/black) to format my Python because I'm lazy. But it can lead to some funny looking things; like I think the `WELL_KNOWN` section could be wide instead of tall.

If you use black you could try to write this in a way so black doesn't make it look goofy. If you don't use black and don't care to and make changes to formatting (I noticed a few more in #4) then that's fine too. (If I want to edit this, I might add `# fmt: off` so it doesn't automatically start rewriting your formatting.)

Reply:
Ah, I don't use black, just `pycodestyle` (which I haven't actually ran yet on #4), but I can format with it at the end, no worries.